packages/page-bounties/src/Bounties.tsx

```diff
@@ -38,16 +38,18 @@ function Bounties (): React.ReactElement {
         empty={bounties && t<string>('No open bounties')}
         header={headerRef.current}
       >
-        {bounties && bestNumber && bounties.map(({ bounty, description, index, proposals }) => (
-          <Bounty
-            bestNumber={bestNumber}
-            bounty={bounty}
-            description={description}
-            index={index}
-            key={index.toNumber()}
-            proposals={proposals}
-          />
-        ))}
+        {bounties && bestNumber &&
+        bounties.sort((bounty1, bounty2) => (bounty2.index.sub(bounty1.index).toNumber()))
```

Comment:
```suggestion
        bounties.sort((a, b) => b.index.cmp(a.index))
```

- Just used the `.cmp`
- Named a/b, the names here are not really important (so slightly shorter without losing context)